GENERAL_TEMPLATES.md

```diff
@@ -0,0 +1,163 @@
+# Templates
+Deliverable
+Documents
+  Test Approach
+   Test Plan
+   Test Schedule
+    Test Specifications
+Test Case / Bug Write-Ups
+  Test Cases / Results
+  Test Coverage Reports
+  GForge Bug tracker for bug reporting
+Reports
+Test results report 
+  Test Final Report - Sign-Off
+
+---
+
+## Image File Naming Best Practices
```

Comment:
"Image File Naming Best Practices” -> Perhaps be done in a pre-commit hook? NPM module to take care of this.

Reply:
Made an [item to work on this](https://github.com/orgs/QueerGlobal/projects/3?pane=issue&itemId=32344223) in the [Infrastructure Roadmap](https://github.com/orgs/QueerGlobal/projects/3).